tasks/install.yml

```diff
@@ -1,21 +1,6 @@
 ---
-- name: install nsupdate which is used by the certbot auth-hook
```

Comment:
This task is completely missing now. Please add it back at least for Debian (and also on CentOS if nsupdate is not installed there per default).

Reply:
Perfect, thanks